modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -461,6 +453,29 @@ public static CatalogSchemaDescriptor schemaOrThrow(Catalog catalog, int schemaI
         return schema;
     }
 
+    /**
+     * Returns schema with given name, or throws {@link CatalogValidationException} if schema with given name not exists.
+     *
+     * @param catalog Catalog to look up schema in.
+     * @param name Name of the schema of interest.
+     * @param shouldThrowIfNotExists Flag indicated should be thrown the {@code CatalogValidationException} for absent schema or just
+     *         return {@code null}.
+     * @return Schema with given name. Never null.
+     * @throws CatalogValidationException If schema with given name is not exists and flag shouldThrowIfNotExists set to {@code true}.
+     */
+    public static CatalogSchemaDescriptor schema(Catalog catalog, String name, boolean shouldThrowIfNotExists)
```

Comment:
```suggestion
    public static @Nullable CatalogSchemaDescriptor schema(Catalog catalog, String name, boolean shouldThrowIfNotExists)
```

also let's fix javadoc